<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="django.contrib.auth.tests.models.ProfileTestCase" tests="1" time="0.001">
	<testcase classname="django.contrib.auth.tests.models.ProfileTestCase" name="test_site_profile_not_available" time="0.001"/>
	<system-out/>
	<system-err/>
</testsuite>
